File: 1961-check-if-string-is-a-prefix-of-array/1961-check-if-string-is-a-prefix-of-array.cpp
class Solution {
public:
    bool isPrefixString(string s, vector<string>& words) {
        string newStr="";
        
        for(string str:words)
        {
            for(char ch: str)
            {
                newStr+=ch;
            }
            if(newStr==s)
            {
                return true;
            }
        }
        return false;
    }
};

File: 1961-check-if-string-is-a-prefix-of-array/1961-check-if-string-is-a-prefix-of-array.java
class Solution {
    public boolean isPrefixString(String s, String[] words) {
        String str="";
        
        for(int i=0; i<words.length; i++)
        {
            if(s.indexOf(words[i])==-1)
            {
                return false;
            }
            str+=words[i];
            if(s.equals(str))
            {
                return true;
            }
        }
        return false;
    }
}